packages/katello/rubygem-anemone/rubygem-anemone.spec

```diff
@@ -58,6 +58,9 @@ cp -a .%{gem_dir}/* \
 %doc %{gem_instdir}/VERSION
 
 %changelog
+* Tue Jun 15 2021 Odilon Sousa <osousa@redhat.com> - 0.7.2-23
+- align version with satellite
```

Comment:
This is not something we have traditionally done in the past as the project and product are different cadences. What would be the reason to start doing so now?

Reply:
the problem is that anemone got a few bumps downstream, and when we rebase a downstream release on upstream, we pull in a "too old" version, thus maybe forgetting changes.

by syncing the release numbers, my intent here was to get out of this mess to have a higher version than downstream.

I know we did that in the past, not often, but we did.